### PR TITLE
Update MKL to 2022.0.0

### PR DESCRIPTION
**Pull request by N5N3**

### CODE REVIEW

File: M/MKL/build_tarballs.jl [hunk 1]
```diff
@@ -13,8 +13,8 @@ platform_sources = [
     (
         platform = Platform("x86_64", "linux"; libc="glibc"),
         source = ArchiveSource(
-            "https://anaconda.org/intel/mkl/2021.1.1/download/linux-64/mkl-2021.1.1-intel_52.tar.bz2",
-            "bfb0fd056576cad99ae1d9c69ada2745420da9f9cf052551d5b91f797538bda2";
+            "https://anaconda.org/intel/mkl/2022.0.1/download/linux-64/mkl-2022.0.1-intel_117.tar.bz2",
```
Comment (giordano) [inline M/MKL/build_tarballs.jl:16]: This is 2022.0.1 instead of 2022.0.0?

Comment (N5N3) [reply M/MKL/build_tarballs.jl:16]: Release notes shows that the lastest version is `2022.0.0`.
But that for `linux32/64` has been marked as `deprecated`.
So I picked the lastest one.

**Comment by ViralBShah** [comment]
We do want the 2022.0.1 for linux, since that has the `_64` symbols for ILP64.

**Comment by ViralBShah** [comment]
I am not sure what is going on here. The PR shows 2022.0.1 for linux, but it appears to have the 2022.0.0.

**Comment by giordano** [comment]
What you're referring to exactly?

**Comment by ViralBShah** [comment]
Never mind. It's all good. It turns out that the new binaries get installed upon invoking `using MKL_jll` the first time. This will let us update LBT.

```
$ nm libmkl_intel_ilp64.so | grep -i _64 | grep -i dpotrf
000000000036f7c0 T dpotrf2_64
000000000036f7c0 T DPOTRF2_64
000000000035f900 T dpotrf_64
000000000035f900 T DPOTRF_64
```

**Pull request merged**